test/integration/functional_test.go

```diff
@@ -573,6 +578,33 @@ func validateMySQL(ctx context.Context, t *testing.T, profile string) {
 	}
 }
 
+// Setup extra file for file sync test (validateFileSync), this
+// need to be done prior to starting up minikube.
+func validateCopySynFile(ctx context.Context, t *testing.T, profile string) {
+	// 1. copy random file to MINIKUBE_HOME/files/etc
+	tempfile := filepath.Join(localpath.MiniPath(), "/files/etc/filesync.txt")
+	err := copy.Copy("./testdata/filesync.txt", tempfile)
+	if err != nil {
+		t.Fatalf("copy: %v", err)
+	}
+}
+
+// validateFileSync start up VM and test sync file between host and VM.
+// On startup Minikube should copy the filesync.txt to /etc folder
+func validateFileSync(ctx context.Context, t *testing.T, profile string) {
+	// check file existence
+	statusCmd := "cat /etc/filesync.txt"
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "-p", profile, "ssh", statusCmd))
+	if err != nil {
+		t.Errorf("%s failed: %v", rr.Args, err)
+	}
+
+	// 'no' result means sync did not happen
+	if strings.Contains(rr.Stdout.String(), "no") {
```

Comment:
Since the previous command succeeded, we know that the file exists. If we're going to check output, let's check it against the original file instead:

```
expected, err := ioutil.ReadFile("./testdata/filesync.txt")

if diff := cmp.Diff(want, rr.Bytes()); diff != "" {
    t.Errorf("/etc/filesync.txt content mismatch (-want +got):\n%s", diff)
}
```